src/conversion/import-properties.ts

```diff
@@ -81,6 +89,11 @@ const importProperties = (
     setValue(propertyTree, "qProperty.qHyperCubeDef.qMeasures", measures);
   }
 
+  if (viewDataMode) {
+    setValue(propertyTree, "qProperty.totals.show", false);
```

Comment:
if `totals: {}` doesn't exist, is it created?

Reply:
I would assume so yes. https://github.com/qlik-trial/conversion/blob/6f6141c69506e2c2afdccf8497471679d686fea4/src/utils/index.js#L43-L64

Reply:
great